tests/my_house_tests.py

```diff
@@ -0,0 +1,170 @@
+import pytest
+
+from factories.expense import ExpenseFactory
+from house.constants import MINE_ADD_EXPENSE_ROUTE, MINE_EXPENSES_TABLE_ROUTE, \
+    MINE_HOUSE_TABLE_ROUTE, MINE_MINE_PAGE_ROUTE, MINE_SIDEBAR_ROUTE, MINE_EDIT_EXPENSE_ROUTE
+from factories.house import HouseFactory
+from house.forms import ExpenseForm
+from expenses.models import Expenses
+from django.template.loader import get_template
+from tests.const import EXPENSE_AMOUNT, EXPENSE_DATE, EXPENSE_CATEGORY, EXPENSE_DESCRIPTION, EXPENSE_BAD_AMOUNT, \
+    EXPENSE_BAD_DESCRIPTION, OTHER_EXPENSE_AMOUNT
+
+
+EXPENSE_FORM_DATA = {'date': EXPENSE_DATE, 'amount': EXPENSE_AMOUNT, 'category': EXPENSE_CATEGORY,
+                     'description': EXPENSE_DESCRIPTION}
+OTHER_EXPENSE_FORM_DATA = {'date': EXPENSE_DATE, 'amount': OTHER_EXPENSE_AMOUNT, 'category': EXPENSE_CATEGORY,
+                           'description': EXPENSE_DESCRIPTION}
+EXPENSE_BAD_FORM_DATA_DESCRIPTION = {'date': EXPENSE_DATE, 'amount': EXPENSE_AMOUNT, 'category': EXPENSE_CATEGORY,
+                                     'description': EXPENSE_BAD_DESCRIPTION}
+EXPENSE_BAD_FORM_DATA_AMOUNT = {'date': EXPENSE_DATE, 'amount': EXPENSE_BAD_AMOUNT, 'category': EXPENSE_CATEGORY,
+                                'description': EXPENSE_DESCRIPTION}
+
+
+@pytest.fixture
+def generate_expense_form():
+    form = ExpenseForm(EXPENSE_FORM_DATA)
+    return form
+
+
+@pytest.fixture
+def generate_other_expense_form():
+    form = ExpenseForm(OTHER_EXPENSE_FORM_DATA)
+    return form
```

Comment:
I don't see any place you are using this fixture, is this for future testing?
if so are we sure we need it as a fixture?